contrib/valgrind.supp

```diff
@@ -41,3 +45,34 @@
    ...
    fun:_ZN7leveldbL14InitDefaultEnvEv
 }
+{
+   Suppress boost::filesystem warning
+   Memcheck:Addr16
+   ...
+   fun:wcsnrtombs
+   ...
+   fun:_ZN5boost10filesystem6detail11unique_pathERKNS0_4pathEPNS_6system10error_codeE
+}
+{
+   Suppress boost::filesystem warning
+   Memcheck:Cond
+   fun:_ZN5boost10filesystem6detail28directory_iterator_incrementERNS0_18directory_iteratorEPNS_6system10error_codeE
+   fun:_ZN5boost10filesystem6detail28directory_iterator_constructERNS0_18directory_iteratorERKNS0_4pathEPNS_6system10error_codeE
+   obj:*/libboost_filesystem.so.*
+}
+{
+   Suppress boost::filesystem warning
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:_Znwm
+   fun:_ZN5boost10filesystem8absoluteERKNS0_4pathES3_
+}
+{
+   Suppress SeedInsecureRand warning
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:_Znwm
+   fun:_Z11LogInstancev
+   ...
+   fun:SeedInsecureRand
```

Comment:
What is the issue? This should be fixed.

Reply:
I guess the issue is generally with `LogInstance` and not `SeedInsecureRand`?

Reply:
True! Fixed! :)